pkg/controller/priorityqueue/metrics.go

```diff
@@ -34,7 +33,6 @@ func newQueueMetrics[T comparable](mp workqueue.MetricsProvider, name string, cl
 		workDuration:            mp.NewWorkDurationMetric(name),
 		unfinishedWorkSeconds:   mp.NewUnfinishedWorkSecondsMetric(name),
 		longestRunningProcessor: mp.NewLongestRunningProcessorSecondsMetric(name),
-		added:                   sets.Set[T]{},
```

Comment:
This secondary bookkeeping was added because originally the queue called `metrics.Add` when the queues Add was called before I realized this is wrong, now that `metrics.add` is only called when an item was newly-added or became ready, this isn't needed anymore